### PR TITLE
Update FlareSolverr plugin

### PR DESCRIPTION
**Pull request by maxlareo**
Update the FlareSolverr plugin following https://github.com/pigfrown/iocage-plugin-flaresolverr/pull/3.

### CODE REVIEW

**Review by sonicaj — APPROVED** [review]
Thank you @maxlareo for your contribution!

**Review by sonicaj — COMMENTED** [review]
BTW CI is failing, can you look at that please before we merge ?

**Comment by maxlareo** [comment]
CI is failing because the related PR in the plugin repo has not been merged yet,we have to wait for the plugin PR to be merged before moving forward with this PR.

**Pull request merged**
